angularjs-portal-frame/src/main/webapp/js/services.js

```diff
@@ -31,10 +31,60 @@ app.factory('miscService', function($http, $window, $location) {
     $window._gaq.push(['_trackEvent', category, action, label]);
   }
 
+    var portletMatchesSearchTerm = function(portlet, searchTerm, opts) {
```

Comment:
This frame code is specific to marketplace. The idea behind the frame is to be generic so other applications can use it. Please move this to somewhere in the marketplace code base.